src/features/regexp-v-flag.md

```diff
@@ -0,0 +1,252 @@
+
+```js
+const re = /^\p{RGI_Emoji}$/v;
+
+// Match an emoji that consists of just 1 code point:
+re.test('⚽'); // '\u26BD'
+// → true ✅
+
+// Match an emoji that consists of multiple code points:
+re.test('👨🏾‍⚕️'); // '\u{1F468}\u{1F3FE}\u200D\u2695\uFE0F'
+// → true ✅
+```
+
+This code snippet refers to the property of strings `RGI_Emoji`, which Unicode defines as “the subset of all valid emoji (characters and sequences) recommended for general interchange”. With this, we can now match emoji regardless of how many code points they consist of under the hood!
+
+The `v` flag enables support for the following Unicode properties of strings from the get-go:
+
+- `Basic_Emoji`
+- `Emoji_Keycap_Sequence`
+- `RGI_Emoji_Modifier_Sequence`
+- `RGI_Emoji_Flag_Sequence`
+- `RGI_Emoji_Tag_Sequence`
+- `RGI_Emoji_ZWJ_Sequence`
+- `RGI_Emoji`
+
+This list of supported properties might grow in the future as the Unicode Standard defines additional properties of strings. Although all current properties of strings happen to be emoji-related, future properties of strings might serve entirely different use cases.
+
+*Note:* Although properties of strings are currently gated on the new `v` flag, [we plan to eventually make them available in `u` mode as well](https://github.com/tc39/proposal-regexp-v-flag/issues/49).
+
+## Set notation + string literal syntax { #set-notation }
+
+When working with `\p{…}` escapes (be it character properties or the new properties of strings) it can be useful to perform difference/subtraction or intersection. With the `v` flag, character classes can now be nested, and those set operations can now be performed within them.
+
+### Difference/subtraction with `--` { #difference }
+
+The syntax `A&&B` can be used to match strings _in `A` but not in `B`_, a.k.a. difference/subtraction. For example, what if you want to match all Greek symbols except for the letter `π`? Previously, there was no way to benefit from `\p{Script_Extensions=Greek}` in this case, since it already matches `π`. The only way to achieve this was to go back to hand-writing a lengthy character class containing the individual characters or ranges. And if Unicode added another Greek character, you would need to notice, and manually update your character class.
```

Comment:
```suggestion
The syntax `A--B` can be used to match strings _in `A` but not in `B`_, a.k.a. difference/subtraction. For example, what if you want to match all Greek symbols except for the letter `π`? Previously, there was no way to benefit from `\p{Script_Extensions=Greek}` in this case, since it already matches `π`. The only way to achieve this was to go back to hand-writing a lengthy character class containing the individual characters or ranges. And if Unicode added another Greek character, you would need to notice, and manually update your character class.
```